risc0/zkvm/platform/src/syscall.rs

```diff
@@ -895,9 +895,8 @@ pub extern "C" fn sys_exit(status: i32) -> ! {
 /// `control_id` must be aligned and dereferenceable.
 ///
 /// `input` must be aligned and have `input_len` u32s dereferenceable
-#[cfg(feature = "export-syscalls")]
+#[cfg(all(feature = "export-syscalls", feature = "unstable"))]
 #[no_mangle]
-#[stability::unstable]
```

Comment:
Why does this need to be changed?

Reply:
This breaks the ability for the `unstable` attribute to adjust the docs with an unstable warning